nautobot/extras/templates/extras/job.html

```diff
@@ -97,7 +103,7 @@ <h1>{{ job_model }}</h1>
                         </div>
                         <div class="pull-right">
                             <button type="submit" name="_run" id="id__run" class="btn btn-primary"
-                                {% if not perms.extras.run_job or not job_model.runnable %}
+                                {% if not perms.extras.run_job or not job_model.runnable or job_model.has_sensitive_variables or job_model.approval_required %}
```

Comment:
This seems wrong - this will prevent any `has_sensitive_variables` *or* `approval_required` job from being run?

Reply:
Can we perhaps add the `has_sensitive_variables and approval_required` check into the `job_model.runnable` logic directly?